index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
```

Comment:
No, because in the case the issuer included structure in the context, and a verifier ignores the context, the verifier ignores the issuers intent.

The verifier has no way to know if the issuer meant for them to ignore the context or not.

Reply:
> No, because in the case the issuer included structure in the context, and a verifier ignores the context, the verifier ignores the issuers intent.

No verifier can ignore `@context`; that is not conformant processing. You can only consume properties from contexts you understand. If that's only the core context, that's fine -- but you can't read properties from other contexts unless you understand them.

Reply:
@OR13, thoughts?

Reply:
Decline to implement feedback, although as I said in the other comments, while the current language is effective, it might be improved in a future editorial PR.